.ci/test_01.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+test_name="$(basename "${0}" '.sh')"
+
+cd "${WORKSPACE}"
```

Comment:
We already use the word "workspace" for another purpose. I used `source` in a similar situation (c.f., https://github.com/HCK-CI/AutoHCK/commit/9726cc0bf31a3d0f513980805a4acfc1defdff51)

Reply:
I can reexport SOURCE=WORKSPACE but this is Jenkins ENV, so it anyway will be present.

Reply:
Perhaps you may rename `.ci` to `jenkins`; "CI" is unclear, considering we already have GitHub Actions. We can refer to the documentation of Jenkins if it's for that platform.